### PR TITLE
Add CMBF model

### PR DESCRIPTION
**Pull request by yangxudong**

### CODE REVIEW

File: easy_rec/python/model/rank_model.py [hunk 1]
```diff
@@ -300,7 +322,7 @@ def _build_distribute_metric_impl(self,
       else:
         raise ValueError('Wrong class number')
     elif metric.WhichOneof('metric') == 'gauc':
-      assert loss_type == LossType.CLASSIFICATION
+      assert loss_type in [LossType.CLASSIFICATION, LossType.F1_REWEIGHTED_LOSS, LossType.PAIR_WISE_LOSS]
```
Comment (lgqfhwy) [inline easy_rec/python/model/rank_model.py:325]: 这个有没有更好的方式来实现，这样每次新加一个 loss 函数，都需要在每个 metric 下面都加一下，感觉有点繁琐

File: easy_rec/python/model/cmbf.py [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+      hidden_size=hidden_size,
+      num_hidden_layers=self._txt_self_attention_layer_num,
+      num_attention_heads=self._head_num,
+      attention_mask=attention_mask,
+      intermediate_size=hidden_size * 2,
+      hidden_dropout_prob=self._model_config.hidden_dropout_prob,
+      attention_probs_dropout_prob=self._model_config.attention_probs_dropout_prob,
+      name='text_self_attention'
+    )  # shape: [batch_size, txt_seq_length, hidden_size]
+    print('txt_attention_fea:', txt_attention_fea.shape)
```
Comment (lgqfhwy) [inline easy_rec/python/model/cmbf.py:239]: print 可以改成 logging 好像更合适一点

File: easy_rec/python/model/rank_model.py [hunk 3]
```diff
@@ -27,7 +27,7 @@ def __init__(self,
                                     labels, is_training)
     self._loss_type = self._model_config.loss_type
     self._num_class = self._model_config.num_class
-
+    self._losses = self._model_config.losses
```
Comment (lgqfhwy) [inline easy_rec/python/model/rank_model.py:30]: 为什么不能用 self._losses 而要额外定义一个 self._losses

File: easy_rec/version.py [hunk 4]
```diff
@@ -1,3 +1,3 @@
 # -*- encoding:utf-8 -*-
 # Copyright (c) Alibaba, Inc. and its affiliates.
-__version__ = '0.4.7'
+__version__ = '0.4.8'
```
Comment (lgqfhwy) [inline easy_rec/version.py:3]: merge 一下 master , 已经有 0.4.8 了。

File: easy_rec/python/protos/feature_config.proto [hunk 5]
```diff
@@ -71,6 +71,8 @@ message FeatureConfig {
 
     // delimeter to separate sequence multi-values
     optional string seq_multi_sep = 101;
+    // truncate sequence data to max_seq_len
+    optional uint32 max_seq_len = 102;
```
Comment (lgqfhwy) [inline easy_rec/python/protos/feature_config.proto:75]: 序列特征不需要这一个值，可以自动算出来的

**Review by lgqfhwy — APPROVED** [review]

File: easy_rec/python/protos/feature_config.proto (hunk 5 above)
Comment (yangxudong) [reply easy_rec/python/protos/feature_config.proto:75]: 这个值是用来截断序列的，如果序列长度很大，则后续模型的计算量会很大。

File: easy_rec/python/model/rank_model.py (hunk 1 above)
Comment (yangxudong) [reply easy_rec/python/model/rank_model.py:325]: 嗯，更好的实现方式可以在后续版本中考虑！

**Review by chengmengli06 — APPROVED** [review]
